### PR TITLE
feat(Aptos): Farm support duo rewards

### PR DESCRIPTION
**Pull request by ChefMomota**
---

### Detailed summary
- Added `CHECK_USER_IP_API` constant to `apps/aptos/config/index.ts`
- Updated `earnLabel` in `apps/web/src/views/Farms/components/FarmCard/FarmCard.tsx`
- Updated imports and constants in `apps/aptos/config/constants/farms/2.ts`
- Reordered imports in `apps/aptos/state/farms/utils/pendingCake.ts`
- Updated `getDisplayApr` function in `apps/aptos/components/Farms/components/getDisplayApr.ts`
- Added `useCheckIsUserIpPass` hook in `apps/aptos/components/Farms/hooks/useCheckIsUserIpPass.ts`
- Updated translations in `packages/localization/src/config/translations.json`
- Added `calcPendingRewardApt` function in `apps/aptos/state/farms/utils/pendingApt.ts`
- Added `fetchAptosView` function in `packages/awgmi/core/src/view/fetchAptosView.ts`
- Updated imports and imports order in `apps/web/src/__tests__/translations.test.ts`
- Updated imports and formatting in `packages/widgets-internal/farm/components/StakedLP/index.tsx`
- Updated imports and formatting in `packages/farms/src/v2/deserializeFarm.ts`
- Updated imports and formatting in `apps/aptos/pages/farms/index.tsx`
- Updated types and formatting in `packages/farms/src/types.ts`
- Updated imports and formatting in `apps/aptos/pages/farms/history.tsx`
- Updated imports and formatting in `apps/aptos/components/Farms/components/FarmTable/Apr.tsx`

> The following files were skipped due to too many changes: `apps/aptos/components/Farms/components/FarmTable/Apr.tsx`, `apps/web/src/views/Farms/components/FarmCard/ApyButton.tsx`, `apps/aptos/components/Farms/components/FarmCard/EarnedTokenInfo.tsx`, `packages/widgets-internal/farm/components/FarmTable/index.tsx`, `packages/awgmi/core/src/index.ts`, `packages/widgets-internal/farm/components/FarmTable/Actions/HarvestAction.tsx`, `apps/aptos/components/Farms/components/FarmCard/CardHeading.tsx`, `packages/awgmi/core/src/index.test.ts`, `packages/uikit/src/components/RoiCalculatorModal/RoiCard.tsx`, `apps/aptos/components/Farms/components/FarmTable/Row.tsx`, `apps/aptos/components/Farms/components/FarmTable/FarmTable.tsx`, `packages/uikit/src/components/RoiCalculatorModal/RoiCalculatorModal.tsx`, `apps/aptos/components/Farms/components/FarmCard/CardActionsContainer.tsx`, `packages/uikit/src/components/RoiCalculatorModal/RoiCalculatorFooter.tsx`, `apps/aptos/components/Farms/components/FarmCard/ApyButton.tsx`, `apps/web/src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx`, `apps/aptos/components/Farms/components/FarmTable/Actions/ActionPanel.tsx`, `apps/aptos/components/Farms/components/FarmCard/EarnedUsdPrice.tsx`, `packages/widgets-internal/farm/components/DepositModal/index.tsx`, `apps/aptos/config/constants/contracts/masterchef.ts`, `apps/aptos/components/Farms/components/Farms.tsx`, `apps/aptos/components/Farms/components/FarmTable/Actions/HarvestAction.tsx`, `apps/web/src/views/Farms/components/FarmCard/StakeAction.tsx`, `apps/aptos/components/Farms/components/FarmTable/Actions/StakedAction.tsx`, `apps/web/src/views/Farms/components/FarmTable/Actions/StakedAction.tsx`, `apps/aptos/components/Farms/components/FarmCard/StakeAction.tsx`, `apps/aptos/components/Farms/components/FarmCard/HarvestAction.tsx`, `apps/aptos/state/farms/hook.ts`, `apps/aptos/components/Farms/components/FarmCard/FarmCard.tsx`, `apps/aptos/config/constants/farms/1.ts`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/aptos/components/Farms/hooks/useCheckIsUserIpPass.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { useQuery } from '@pancakeswap/awgmi'
+import { CHECK_USER_IP_API } from 'config/index'
+
+export const useCheckIsUserIpPass = (): boolean => {
+  const { data } = useQuery(['checkIsUserIpPass'], async () => {
```
Comment (chef-jojo) [inline apps/aptos/components/Farms/hooks/useCheckIsUserIpPass.ts:5]: Let's check when user is connected. and make it not revalidate that across the session

File: apps/aptos/config/coins.ts [hunk 2]
```diff
@@ -52,7 +52,8 @@ export const CAKE = {
   ),
   [ChainId.TESTNET]: new Coin(
     ChainId.TESTNET,
-    '0xe0e5ad285cbcdb873b2ee15bb6bcac73d9d763bcb58395e894255eeecf3992cf::pancake::Cake',
+    '0x48e0e3958d42b8d452c9199d4a221d0d1b15d14655787453dbe77208ced90517::coins::CAKE',
+    // '0xe0e5ad285cbcdb873b2ee15bb6bcac73d9d763bcb58395e894255eeecf3992cf::pancake::Cake',
```
Comment (ChefMomota) [inline apps/aptos/config/coins.ts:56]: Revert before merge

File: apps/aptos/config/constants/contracts/masterchef.ts [hunk 3]
```diff
@@ -1,7 +1,8 @@
 /* eslint-disable camelcase */
 import { Types } from 'aptos'
 
-export const ADDRESS = '0x7968a225eba6c99f5f1070aeec1b405757dee939eabcfda43ba91588bf5fccf3' as const
+// export const ADDRESS = '0x7968a225eba6c99f5f1070aeec1b405757dee939eabcfda43ba91588bf5fccf3' as const
+export const ADDRESS = '0x4886f00f0167b6e02a933a771000c668c1d5496feb6c5a992fa9f3da6de9317c' as const
```
Comment (ChefMomota) [inline apps/aptos/config/constants/contracts/masterchef.ts:5]: Revert before merge

File: apps/aptos/config/constants/contracts/masterchef.ts [hunk 4]
```diff
@@ -1,7 +1,8 @@
 /* eslint-disable camelcase */
 import { Types } from 'aptos'
 
-export const ADDRESS = '0x7968a225eba6c99f5f1070aeec1b405757dee939eabcfda43ba91588bf5fccf3' as const
+// export const ADDRESS = '0x7968a225eba6c99f5f1070aeec1b405757dee939eabcfda43ba91588bf5fccf3' as const
+export const ADDRESS = '0xa7aa53dc34fdbe3198f57e0c86f0b41d70a30a318bb5b18318ef764892661e0a' as const
```
Comment (ChefMomota) [inline apps/aptos/config/constants/contracts/masterchef.ts:5]: Revert before merge

File: apps/aptos/config/constants/farms/1.ts
Comment (ChefMomota) [inline apps/aptos/config/constants/farms/1.ts:1]: Revert before merge

File: apps/aptos/config/constants/priceHelperLps/farms/1.ts [hunk 5]
```diff
@@ -34,6 +34,14 @@ const priceHelperLps: Omit<SerializedFarmConfig, 'pid'>[] = [
...
     token: mainnetTokens.etern,
   },
+  {
+    pid: null,
+    lpSymbol: 'APT-lzUSDC LP',
+    lpAddress:
+      '0xc7efb4076dbe143cbcd98cfaaa929ecfc8f299203dfff63b95ccb6bfe19850fa::swap::LPToken<0x1::aptos_coin::AptosCoin, 0xf22bede237a07e121b56d91a491eb7bcdfd1f5907926a9e58338f964a01b17fa::asset::USDC>',
+    token: mainnetTokens.lzusdc,
+    quoteToken: mainnetTokens.apt,
+  },
```
Comment (ChefMomota) [inline apps/aptos/config/constants/priceHelperLps/farms/1.ts:44]: Revert before merge

File: apps/aptos/config/constants/contracts/masterchef.ts [hunk 6]
```diff
@@ -1,7 +1,8 @@
 /* eslint-disable camelcase */
 import { Types } from 'aptos'
 
-export const ADDRESS = '0x7968a225eba6c99f5f1070aeec1b405757dee939eabcfda43ba91588bf5fccf3' as const
+// export const ADDRESS = '0x7968a225eba6c99f5f1070aeec1b405757dee939eabcfda43ba91588bf5fccf3' as const
+export const ADDRESS = '0x7c00653b67c9a7b65a4318a7380545e77b1850ea332f91e44474b9bb5a393570' as const
```
Comment (ChefMomota) [inline apps/aptos/config/constants/contracts/masterchef.ts:5]: Revert before merge

File: apps/aptos/config/constants/farms/1.ts
Comment (ChefMomota) [inline apps/aptos/config/constants/farms/1.ts:1]: Revert before merge

**Review by chefjackson — APPROVED** [review]

**Pull request merged**